kubevirtci

```diff
@@ -0,0 +1,415 @@
+	  clusterctl <clusterctl options>   Run clusterctl commands against the cluster
+
+	  install-capk                      Installs capk from published release manifests
+	  install-metallb                   Installs metallb into the infra cluster
+	  install-calico                    Installs calico cni into tenant cluster
+	  curl-lb <lb name> [lb namespace]  Curls lb service within infra cluster
+
+	  ssh-infra <node name>             SSH into one of the infra nodes (like node01)
+	  ssh-tenant <vmi> [vmi namespace]  SSH into one of the guest nodes
+	  create-cluster                    Create new kubernetes tenant cluster
+	  create-external-cluster           Create new kubernetes tenant cluster simulated as running on external infra
+	  destroy-cluster                   Destroy the tenant cluster
+
+	  help                              Print usage
+	"
+}
+
+function kubevirtci::kubeconfig() {
+	cluster-up/cluster-up/kubeconfig.sh
+}
+
+function kubevirtci::fetch_kubevirtci() {
+	[[ -d cluster-up ]] || git clone https://github.com/kubevirt/kubevirtci.git cluster-up
+	(cd cluster-up && git checkout main > /dev/null 2>&1 && git pull > /dev/null && git checkout ${KUBEVIRTCI_TAG} > /dev/null 2>&1)
+	mkdir -p ./hack/tools/bin/
+	if [ ! -f "${_default_clusterctl_path}" ]; then
+		echo >&2 "Downloading clusterctl version ${CLUSTERCTL_VERSION}..."
+		curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/${CLUSTERCTL_VERSION}/clusterctl-linux-amd64 -o ${_default_clusterctl_path}
+		chmod u+x ${_default_clusterctl_path}
+	fi
+	if [ ! -f "${_default_virtctl_path}" ]; then
+		echo >&2 "Downloading virtctl ..."
+		curl -L https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/virtctl-${KUBEVIRT_VERSION}-linux-amd64 -o ${_default_virtctl_path}
+		chmod u+x ${_default_virtctl_path}
+	fi
+}
+
+function kubevirtci::up() {
+	make cluster-up -C cluster-up
+	export KUBECONFIG=$(cluster-up/cluster-up/kubeconfig.sh)
+	echo "installing kubevirt..."
+	${_kubectl} apply -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-operator.yaml
+	${_kubectl} apply -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml
+	echo "installing capi..."
+
+	cat << EOF > ${_default_bin_path}/clusterctl_config.yaml
+---
+cert-manager:
+  url: "https://github.com/cert-manager/cert-manager/releases/latest/cert-manager.yaml"
+EOF
+	$CLUSTERCTL_PATH init -v 4 --config=${_default_bin_path}/clusterctl_config.yaml
+	echo "waiting for kubevirt to become ready, this can take a few minutes. You can safely abort this step, the cluster is ready ..."
+	${_kubectl} -n kubevirt wait kv kubevirt --for condition=Available --timeout=5m
+}
+
+function kubevirtci::down() {
+	make cluster-down -C cluster-up
+}
+
+function kubevirtci::build() {
+	export REGISTRY="127.0.0.1:$(cluster-up/cluster-up/cli.sh ports registry)"
+	make docker-build
+	make docker-push
```

Comment:
you'll want to have `make` targets that build the csi container image and can push it to the kubevirtci repo

Reply:
ok, I need to use these two targets: https://github.com/kubevirt/csi-driver/pull/40